lua/flatten/guest.lua

```diff
@@ -73,14 +83,32 @@ M.init = function(host_pipe)
 	vim.api.nvim_create_autocmd("BufEnter", {
 		pattern = "*",
 		callback = function()
+			files = vim.tbl_map(function(bufnr)
+				local name = vim.fn.bufname(bufnr)
```

Comment:
vim.api.nvim_buf_get_name(bufnr) - the main reason I wasn't doing this is that we need to do a number of additional checks here. 

Some important ones are:
- Ensuring the buffers are valid and listed
- Ensuring the buffers are of type `` or `acwrite`

Reply:
I dont know if there is another way to solve #52? atleast I think doing those checks is easier than any other fix

Reply:
Added those checks